packages/alchemy/src/signer/client/index.ts

```diff
@@ -27,41 +29,35 @@ export const AlchemySignerClientParamsSchema = z.object({
     iframeContainerId: z.string(),
   }),
   rpId: z.string().optional(),
+  rootOrgId: z
+    .string()
+    .optional()
+    .default("24c1acf5-810f-41e0-a503-d5d13fa8e830"),
```

Comment:
what is this default value? our org id?

Reply:
this is our default prod org, yea. it's necessary for certain whoami requests and we always had it set, but now we pass it as a parameter instead